desktop/ui/src/main/java/org/datacleaner/windows/AboutDialog.java

```diff
@@ -107,7 +107,7 @@ protected int getDialogWidth() {
 
     @Override
     protected boolean isWindowResizable() {
-        return true;
+        return false;
```

Comment:
Made me take another look, though. We should probably have a little bit bigger window for the sake of the licenses.